rust/private/rustc.bzl

```diff
@@ -194,6 +194,7 @@ def rustc_compile_action(
         dep_info.transitive_libs +
         [toolchain.rustc] +
         toolchain.crosstool_files +
+        ctx.files._cc_toolchain +
```

Comment:
Add a comment explaining the why of this addition and with a TODO to find a canonical way. From what I read from this doc theses files should be included in crosstool_files.

side-reflection: this is a pretty huge sum of lists. I am not sure about the resulting complexity (java linked list should be fine but IIRC those are ArrayList, who have O(n+m) complexity). Also the transitive_libs can become very huge, this could be a bottle neck on large list of compilation.

Reply:
Yup I second @damienmg's comment, make a good use of the `transitive` attribute of depset https://docs.bazel.build/versions/master/skylark/lib/globals.html#depset :)